omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/FormResourceController1_9Test.java

```diff
@@ -176,6 +176,18 @@ public void shouldGetAFormResourceByUuid() throws Exception {
 		assertEquals(resource.getName(), PropertyUtils.getProperty(result, "name"));		
 	}
 	
+	@Test
+	public void shouldEditAFormResource() throws Exception {		
+		final String EDITED_NAME = "Edited Form Resource";
+		String json = "{ \"name\":\"" + EDITED_NAME + "\" }";
+		MockHttpServletRequest req = request(RequestMethod.POST, getURI() + "/" + getUuid());
+		req.setContent(json.getBytes());
+		handle(req);
+
+		FormResource editedForm = formService.getFormResourceByUuid(getUuid());
+		Assert.assertEquals(EDITED_NAME, editedForm.getName());
```

Comment:
Can you assert that the name was not like this before the edit?

Reply:
@dkayiwa take a look and verify whether the changes are sufficient!